proto/common/dpauth.proto

```diff
@@ -5,14 +5,15 @@ package proto;
 option go_package = ".;proto";
 
 service DPAuthService{
-  rpc GetWorkspaceToken( GetWorkspaceTokenRequest ) returns ( GetWorkspaceTokenResponse );
+  rpc GetConnectionIdentifier( GetConnectionIdentifierRequest ) returns ( GetConnectionIdentifierResponse );
```

Comment:
This is going to break in terms of backward compatibility. Because we will be going to update the cp-router first. Also, all data planes will not be updated at once. 
I guess we need to keep the existing implementation as well for Backward compatibility.

Reply:
Added backwards compatibility. Can you review again?